modules/tests/shared/src/test/scala/lucuma/core/util/GidSuite.scala

```diff
@@ -27,6 +29,7 @@ final class GidSuite extends DisciplineSuite {
   checkAll("Gid[Id]", OrderTests[Id].order)
   checkAll("Gid[Id].isoPosLong", IsoTests(Gid[Id].isoPosLong))
   checkAll("Gid[Id].fromString", PrismTests(Gid[Id].fromString))
+  checkAll("Gid[Id]", CodecTests[Id].unserializableCodec)
```

Comment:
Why not `CodecTest[Gid[Id]]`

Reply:
We're testing the codec for `Id`, same as `OrderTest` above.

Reply:
So `OrderTests` and `CodecTests` are checking that the derived `Order` and `Encoder/Decoder` instances for `Id` are correct.

Reply:
Ok, now I got it, thanks